.github/workflows/main.yml

```diff
@@ -17,3 +17,9 @@ jobs:
 
       - name: Install
         run: yarn install
+
+      - name: Install markdownlinter-cli
+        run: npm install markdownlint-cli
```

Comment:
With this being a Node project we could probably remove this from the CI and have it as a project dependency like eslint. Thoughts?

Reply:
Yes, I am in the process of adding a card to Trello to do that very thing.